charts/sn-platform/values.yaml

```diff
@@ -2124,12 +2124,7 @@ vault:
       statsd_address: ""
     bankVaults:
       probe:
-        readinessProbe:
```

Comment:
can we just comment out the block instead of removing it? else user might have to look into the template to know how to add redinessProbe

Reply:
sounds reasonable.